django/forms/widgets.py

```diff
@@ -49,10 +49,8 @@ def __init__(self, media=None, css=None, js=None):
                 css = {}
             if js is None:
                 js = []
-
-        # create copies
-        self._css = dict(css)
```

Comment:
I'm okay keeping the copying stuff if we have a test to show why they're needed. Also, `dict()` might need to be `deepcopy` instead.